File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    # Open page
    driver.get("https://training-support.net/webelements/dynamic-controls")

    print("Page title:", driver.title)

    checkbox = driver.find_element(By.ID, "checkbox")

    print("Checkbox initially selected:", checkbox.is_selected())

    checkbox.click()

    print("Checkbox after click:", checkbox.is_selected())

    checkbox.click()

    print("Checkbox after second click:", checkbox.is_selected())

File: Selenium/Activities/Activity6.java
package examples;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity6 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println("Page title: " + driver.getTitle());

        WebElement checkbox = driver.findElement(By.id("checkbox"));

        System.out.println("Checkbox initially selected: " + checkbox.isSelected());

        checkbox.click();

        System.out.println("Checkbox after click: " + checkbox.isSelected());

        checkbox.click();

        System.out.println("Checkbox after second click: " + checkbox.isSelected());

        driver.quit();
    }
}